.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
+    "singleQuote": true,
```

Comment:
Are we using single or double?

Reply:
I like to use single but I'm with using ether one